src/main/resources/assets/actuallyadditions/lang/en_US.lang

```diff
@@ -579,6 +581,10 @@ tooltip.actuallyadditions.battery.changeMode=Sneak-right-click to toggle.
 tooltip.actuallyadditions.previouslyDoubleFurnace=Previously "Double Furnace"
 tooltip.actuallyadditions.previouslyBag=Previously "Bag"
 tooltip.actuallyadditions.previouslyVoidBag=Previously "Void Bag"
+tooltip.actuallyadditions.item_filling_wand.selectedBlock=Selected Block: %s
+tooltip.actuallyadditions.item_filling_wand.selectedBlock.none=None
+tooltip.actuallyadditions.itemEnergy=%s/%s Crystal Flux
```

Comment:
I just think it's weird to change such a name, especially because I believe that "Redstone Flux" and "Millibuckets" and stuff like that usually don't get changed either.